sol/src/sol.module.ts

```diff
@@ -14,15 +14,13 @@ import { TerminalComponent } from './terminal/terminal.component'
     SolComponent,
     TerminalComponent
   ],
-  imports: [
-    HttpClientModule,
-    CommonModule,
-    BrowserModule
-  ],
   exports: [
     SolComponent
   ],
-  schemas: [CUSTOM_ELEMENTS_SCHEMA]
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Do we still need CUSTOMS_ELEMENTS_SCHEMA?

Reply:
done